requests.go

```diff
@@ -101,22 +111,30 @@ type mirrorInfo struct {
 	ContentLength int64
 }
 
-func getMirrorInfo(ctx context.Context, client *http.Client, url string) (*mirrorInfo, error) {
-	req, err := http.NewRequest("HEAD", url, nil)
+func (pget *Pget) getMirrorInfo(ctx context.Context, client *http.Client, url string) (*mirrorInfo, error) {
+	req, err := http.NewRequest("GET", url, nil)
```

Comment:
It should be HEAD because the server respond any contents if GET.
we do not treat the content here.

Reply:
Yes, this is true, but I think not all services may support HEAD requests, but I think all services support GET requests, and using GET makes programs more compatible.

Reply:
```suggestion
	req, err := http.NewRequest("HEAD", url, nil)
```